app/views/shared/_notification.html.erb

```diff
@@ -5,13 +5,13 @@
     <% actions = options[:auto_dismiss] ? "animationend->element-removal#remove" : "" %>
     <% animation = options[:auto_dismiss] ? "animate-[appear-then-fades_5s_300ms_both]" : "animate-[appear_5s_300ms_both]" %>
     <%= content_tag :div,
-      class: "max-w-80 bg-white shadow-xs border border-alpha-black-50 border-solid py-3 px-4 rounded-[10px] text-sm font-medium flex gap-4 #{animation}",
+      class: "max-w-80 bg-white shadow-xs border border-alpha-black-50 border-solid py-4 px-4 rounded-[10px] text-sm font-medium flex gap-4 #{animation} group",
```

Comment:
The close button will show when you hover over the outer most div of the notification.